manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/plugins/ServiceTypeHandler.java

```diff
@@ -58,18 +58,11 @@
     private K8sServiceType getServiceType(Boolean external) {
 
         if (external) {
-            if (checkForLoadBalancerType()) {
-                return K8sServiceType.LOAD_BALANCER;
-            } else {
-                return K8sServiceType.NODE_PORT;
-            }
+            // TODO add K8sServiceType.NODE_PORT;
+            return K8sServiceType.LOAD_BALANCER;
         } else {
             return K8sServiceType.CLUSTER_IP;
         }
     }
 
-    private boolean checkForLoadBalancerType() {
```

Comment:
Keep this code, because the ServiceType has to be returned based on the cluster type.